### DISCUSSION TITLE
[Item]_论文构思模板

### DISCUSSION THREAD

**Discussion by XL-Qing** [windingwind/zotero-better-notes#599] [Note Templates]
当看完相关文献后会存在部分闪光点，本模板用作记录当时的头脑风暴，为后续论文撰写提供方便。
![18beae16e326987a561b3cad3e80b1e](https://github.com/windingwind/zotero-better-notes/assets/129154095/f33a1685-3d95-4c1c-aad0-7d6c7d91d023)

模板建好后**需自己手动输入所需笔记**，相关文献引入方法如下：
![ebe98c2b255185daef4971b680a1865](https://github.com/windingwind/zotero-better-notes/assets/129154095/05aa7f53-86a4-489c-ac1e-d4668906baad)

说明：

本模板基于https://github.com/windingwind/zotero-better-notes/discussions/501 修改
当前模板在Zotero7上运行正常
代码自取
```<!-- author:UESTC_Qing Xiaolong-->
<span style="color:#E91E63"><h1><font size="6">论文构思：</font></h1></span>
<span style="color:#2F2F4F"><strong>👨‍🎓作者：</u></strong></span><br/>
<span style="color:#2F2F4F">📅创建时间：2023-<br/>
<span style="color:#2F2F4F; background-color:#ADEAEA"><strong>🔢意向期刊：<br/>
</span >🎯参考文献</strong>：
</a><br/>
</span></h1>\n <h2 style="color:  #FFA500; background-color:  #FFF8DC;">📜 研究主题</h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color:#FF3030; background-color:    #FFF5EE;">🔬 研究方法&设计</h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color:#698B22; background-color:    #F1F8E9;">🚩 预期结果</h2> <hr/>
<p></p>
</span></h1>\n <h2 style="color: #006064; background-color:   #E0F7FA;">📌 讨论&解释</h2> <hr/>
<p></p>
<h3>  <span style="color: #9A32CD; background-color:    #EED2EE;">❓待解决点：</span></h3>
<p></p>
<h3>  <span style="color: #EE1289; background-color:    #FFE1FF;">✨创新点：</span></h3>
<p></p>
</span></h1>\n <h2 style="color:#FF8C00; background-color:   #FFE7BA;">🤔 其他思考</h2> <hr/>

**Comment by Drxiang1** [comment]
代码最后的三个引号要删去，否则预览显示错误

**Reply by yywharbineu** [reply]
请问是哪三个？

**Comment by XL-Qing** [comment]
收到，谢谢，马上修改
        
        
            
        
    
    
    

    

        
  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2023年8月16日 22:21
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              Re: [windingwind/zotero-better-notes] [Item]_论文构思模板 (Discussion #599)
        
      
    
  

代码最后的三个引号要删去，否则预览显示错误

**Comment by XL-Qing** [comment]
错误的符号已经删掉，可以直接复制粘贴就行
        
  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2023年8月19日 07:08
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              Re: [windingwind/zotero-better-notes] [Item]_论文构思模板 (Discussion #599)
        
      
    
  

请问是哪三个？
